test/modules/packages/Crypto/saru/kdf/pbkdf2-2.skipif

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+reqMajor=1
+reqMinor=0
```

Comment:
Can this and one of the other ones symlink to a real one?

Reply:
Sounds good.